qemu_v8.mk

```diff
@@ -189,6 +274,35 @@ soc-term:
 soc-term-clean:
 	$(MAKE) -C $(SOC_TERM_PATH) clean
 
+################################################################################
+# mkimage - create images to be loaded by U-boot
+################################################################################
+# Without the objcopy, the uImage will be 10x bigger.
+uimage: linux $(BINARIES_PATH)
```

Comment:
Why not make the target `$(KERNEL_UIMAGE)`? It seems that `uimage` will always be out of date.